landoapi/api/stacks.py

```diff
@@ -97,6 +102,12 @@ def get(revision_id):
         summary = PhabricatorClient.expect(fields, "summary")
         bug_id = get_bugzilla_bug(revision)
         reviewers = get_collated_reviewers(revision)
+
+        # The sec-approval group must not appear in the commit message reviewers list.
+        # Bug 1590225
+        if sec_approval_project_phid in reviewers:
+            del reviewers[sec_approval_project_phid]
+
         accepted_reviewers = [
             reviewer_identity(phid, users, projects).identifier
             for phid, r in reviewers.items()
```

Comment:
@smacleod I'm concerned by the code duplication here.  The same ~15 lines of code to gather contextual information to build a commit message appears in `lando.api.stacks` and `lando.api.transplants`.  Is it time to refactor this into a common routine?

Reply:
I'd prefer if you leave as-is for now, there's other changes going on in the area and it's not a huge duplication at this point.